nuntium/tests/writeitinstances_test.py

```diff
@@ -372,3 +378,25 @@ def test_no_form_on_homepage_of_empty_instance(self):
         response = self.client.get(url)
 
         self.assertIn('there is no-one to write to', response.content)
+
+    def test_there_is_a_post_submission_url(self):
+        '''There is a post submission URL'''
+        message = self.writeitinstance1.message_set.get(id=1)
+        url = reverse('post_submission', kwargs={
+            'instance_slug': self.writeitinstance1.slug,
+            'slug': message.slug,
+            })
+        self.assertTrue(url)
+
+    def test_get_post_submission_url(self):
+        '''Get a post submission url'''
+        message = self.writeitinstance1.message_set.get(id=1)
+        url = reverse('post_submission', kwargs={
+            'instance_slug': self.writeitinstance1.slug,
+            'slug': message.slug,
+            })
+        response = self.client.get(url)
+        self.assertEquals(response.status_code, 200)
+        self.assertIn('message', response.context)
+        self.assertEquals(response.context['message'], message)
+        self.assertTemplateUsed(response, 'nuntium/message/post_submission.html')
```

Comment:
I think it would be nice to include a test which uses the test client to post to create a new message and checks for the redirect.